src/foam/u2/view/RichChoiceView.js

```diff
@@ -374,7 +377,7 @@ foam.CLASS({
                   return Promise.all(promiseArray).then((resp) => {
                     var index = 0;
                     return this.E().forEach(sections, function(section) {
-                      this
+                      this.addClass(this.myClass('setAbove'))
```

Comment:
```suggestion
                      this.addClass(self.myClass('setAbove'))
```

whoops, should probably be `self`.